build-deno

```diff
@@ -15,7 +15,7 @@ This is a build produced from https://github.com/openai/openai-node – please g
 Usage:
 
 \`\`\`ts
-import OpenAI from "$(echo 'https://deno.land/x/openai@v4.17.0/mod.ts' | sed -E s/@\.+\\//@"$PACKAGE_VERSION"\\//)";
+import OpenAI from "$(echo 'https://deno.land/x/openai@v4.17.1/mod.ts' | sed -E s/@\.+\\//@"$PACKAGE_VERSION"\\//)";
```

Comment:
This should say 4.17.2 @rattrayalex

Reply:
the sed should do that for us but I can clean this up.

Reply:
Ah sorry I read this as the README, this is fine